### PR TITLE
FEAT: Update button file

### PR DESCRIPTION
**Pull request by React-Developer-Usama**
Update Button File

### CODE REVIEW

**Comment by React-Developer-Usama** [comment]
HI New Branch has been Push Kindly review It

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ const Button = (props) => {
   return (
     <>
       <div className="btn">
-        <button>{props.name}</button>
+        {/* Button Type and Text */}
+        <button type={props.type}>{props.name}</button>
```
Comment (PixAbdulBasit) [inline src/components/Button/Button.jsx:9]: Please Make it object destructor. It will More Help Us

**Review by PixAbdulBasit — COMMENTED** [review]
Complete It And Push

**Review by SoftwareEngr-AbdulBasit — COMMENTED** [review]
Done

**Pull request merged**